graphql/phoneVerification/resolver.ts

```diff
@@ -96,6 +96,7 @@ const phoneVerificationResolver = {
 
       // If three codes were already sent, wait for 4h before sending another.
       if (
+        process.env.SMS_ENABLED === '1' &&
```

Comment:
the phone verification attempts was just to reduce sending SMS, if we don't have SMS enabled it doesn't make sense to prevent this, and also for dev and preview deployments should be off too.